### PR TITLE
Add retries for resource tagging

### PR DESCRIPTION
**Pull request by deliahu**

### CODE REVIEW

File: cli/cmd/cluster.go [hunk 1]
```diff
@@ -737,14 +730,30 @@ func createBucketIfNotFound(awsClient *aws.Client, bucket string) error {
 			fmt.Print("\n\n")
 			return err
 		}
-		fmt.Println(" ✓")
 	} else {
-		fmt.Println("￮ using existing s3 bucket:", bucket, "✓")
+		fmt.Print("￮ using existing s3 bucket:", bucket)
 	}
+
+	numTaggingAttempts := 0
```
Comment (vishalbollu) [inline cli/cmd/cluster.go:737]: nit: any reason not to use `for i := 0; i++; i < 10` style for loop here?

File: cli/cmd/cluster.go [hunk 2]
```diff
@@ -756,11 +765,26 @@ func createLogGroupIfNotFound(awsClient *aws.Client, logGroup string) error {
...
 			return err
 		}
-		fmt.Println(" ✓")
 	} else {
-		fmt.Println("￮ using existing cloudwatch log group:", logGroup, "✓")
+		fmt.Print("￮ using existing cloudwatch log group:", logGroup)
 	}
 
+	numTaggingAttempts := 0
+	for true {
```
Comment (vishalbollu) [inline cli/cmd/cluster.go:773]: Same comment as above

File: cli/cmd/cluster.go (hunk 1 above)
Comment (vishalbollu) [inline cli/cmd/cluster.go:737]: drop a comment explaining why retries may be necessary

Comment (deliahu) [reply cli/cmd/cluster.go:737]: No particular reason, that was just how it occurred to me to implement it. How does this one look?

```go
for i := 0; i < 10; i++ {
	err = awsClient.TagLogGroup(logGroup, tags)
	if err == nil {
		fmt.Println(" ✓")
		return nil
	}
	if !aws.IsErrCode(err, cloudwatchlogs.ErrCodeResourceNotFoundException) {
		break
	}
}

fmt.Print("\n\n")
return err
```

**Pull request merged**